.env.example

```diff
@@ -1,7 +1,9 @@
+ARCANA_DASHBOARD=
+ARCANA_DOCS=
+BTW_SO_ENABLED=false
```

Comment:
is the value for `BTW_SO_ENABLED` intentional in example file?

can we add `URL` in suffix of `ARCANA_DASHBOARD` and `ARCANA_DOCS`

Reply:
Don't recall why we set it to false. Must have been causing an issue during development.

Added URL suffixes.

Reply:
I mean example file usually don't have values

Reply:
Removed.